src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -4464,6 +4464,312 @@ Activate by holding the key for the character you want to add an accent to, then
+    <value>100%</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition.Header" xml:space="preserve">
+    <value>Timer Position</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_TopLeftCorner.Content" xml:space="preserve">
+    <value>Top left corner</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_TopCenter.Content" xml:space="preserve">
+    <value>Top center</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_TopRightCorner.Content" xml:space="preserve">
+    <value>Top right corner</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_Left.Content" xml:space="preserve">
+    <value>Left</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_Center.Content" xml:space="preserve">
+    <value>Center</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_Right.Content" xml:space="preserve">
+    <value>Right</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_BottomLeftCorner.Content" xml:space="preserve">
+    <value>Bottom left corner</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_BottomCenter.Content" xml:space="preserve">
+    <value>Bottom center</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_BottomRightCorner.Content" xml:space="preserve">
+    <value>Bottom right corner</value>
+  </data>
+  <data name="ZoomIt_Break_ShowBackgroundBitmap.Header" xml:space="preserve">
+    <value>Show Background Bitmap</value>
+  </data>
+  <data name="ZoomIt_Break_ShowFadedDesktop.Content" xml:space="preserve">
+    <value>Use faded desktop as background</value>
+  </data>
+  <data name="ZoomIt_Break_ShowImageFile.Content" xml:space="preserve">
+    <value>Use image file as background</value>
+  </data>
+  <data name="ZoomIt_Break_BackgroundFile.Header" xml:space="preserve">
+    <value>Background Image File</value>
+  </data>
+  <data name="ZoomIt_Break_BackgroundFile_BrowseButton.Content" xml:space="preserve">
+    <value>Browse</value>
+  </data>
+  <data name="ZoomIt_Break_BackgroundFile_Picker_Dialog_Title" xml:space="preserve">
+    <value>Specify background file...</value>
+  </data>
+  <data name="FilePicker_ZoomIt_BitmapFilesFilter" xml:space="preserve">
+    <value>Bitmap Files</value>
+  </data>
+  <data name="FilePicker_ZoomIt_AllPicturesFilter" xml:space="preserve">
+    <value>All Picture Files</value>
+  </data>
+  <data name="ZoomIt_Break_BackgroundStretch.Header" xml:space="preserve">
+    <value>Scale to screen</value>
+  </data>
+  <data name="ZoomIt_RecordGroup.Header" xml:space="preserve">
+    <value>Record</value>
+  </data>
+  <data name="ZoomIt_RecordGroup.Description" xml:space="preserve">
+    <value>Record video of the unzoomed live screen or a static zoomed session by entering the recording hot key and finish the recording by entering it again.
+
+To crop the portion of the screen that will be recorded, enter the hotkey with the Shift key in the opposite mode.
```

Comment:
To finish recording you need to enter "Record video" (ctrl+5) hotkey, not (ctrl+shift+5). Might be worth pointing out, make it clear.

Also, same Q - what if I set hotkey that uses Shift or Alt?

Reply:
> To finish recording you need to enter "Record video" (ctrl+5) hotkey, not (ctrl+shift+5). Might be worth pointing out, make it clear.

or is this a bug? Because alt+ctrl+5 both starts and ends recording of the window

Reply:
let me rephrase:
 - if you enter crop and record mode (Ctrl + Shift + 5) , same hotkey (Ctrl + Shift + 5) does not end it. You have to pres Ctrl + 5
 
 This is not the case for window record mode (Ctrl + Alt + 5). You end it with Ctrl + Alt + 5